tests/e2e/deployment/deployment_test.go

```diff
@@ -194,4 +196,48 @@ var _ = Describe("Application deployment test in E2E scenario", func() {
 			utils.CheckPodDeleteState(ctx.Cfg.K8SMasterForKubeEdge+constants.AppHandler, podlist)
 		})
 	})
+
+	Context("Test Edge Autonomy", func() {
+		var podName string
+		BeforeEach(func() {
+			// Get current test description
+			testDescription = CurrentGinkgoTestDescription()
+			// Start test timer
+			testTimer = DeploymentTestTimerGroup.NewTestTimer(testDescription.TestText)
+		})
+		AfterEach(func() {
+			// End test timer
+			testTimer.End()
+			// Print result
+			testTimer.PrintResult()
+			// run tests/e2e/scripts/autonomy.sh to test autonomy function
+
+			gopath, err := utils.GetGOPATH()
+			if err != nil {
+				utils.Fatalf("failed to get GOPATH: %v", err)
+				return
+			}
+			utils.Infof("gopath is %s", gopath)
+
+			script := path.Join(gopath, "src/github.com/kubeedge/kubeedge/tests/e2e/scripts/autonomy.sh")
+			utils.Infof("autonomy scripts path is %v", script)
```

Comment:
failed to locate the correct path of the shell script. and ref the CI result, the log is 
```
Nov  1 02:54:37.594: Info: gopath is /home/runner/go
Nov  1 02:54:37.595: Info: autonomy scripts path is /home/runner/go/src/github.com/kubeedge/kubeedge/tests/e2e/scripts/autonomy.sh
Nov  1 02:54:37.598: Info: autonomy.sh exec output is

Nov  1 02:54:37.598: Fatal: exec autonomy scripts failed: exit status 127
```
but the code path is 
`/home/runner/work/kubeedge/kubeedge/tests/e2e/deployment/deployment_test.go:230`

code is not in the `GOPATH` dir, I think this method it not correct.
It's easy to locate the go source code dir in the script, but it's not easy to locate the shell script path in the go source code directory.